docs/source/best-practices.rst

```diff
@@ -369,5 +369,3 @@ Compute related results with shared computations in a single :func:`dask.compute
 
 This allows Dask to compute the shared parts of the computation (like the
 ``dd.read_csv`` call above) only once, rather than once per ``compute`` call.
-For more guidance on when to call ``compute`` when working with Dask DataFrames
-or Arrays see `this blog post <https://www.coiled.io/blog/dask-compute>`_.
```

Comment:
Coiled removed this post. After reading the archived version again, I don't think it adds much. I considered linking to https://distributed.dask.org/en/stable/manage-computation.html#dask-collections-to-concrete-values, but was worried it'd be throwing new folks into the deep end.